MediaRecorder.html

```diff
@@ -462,8 +462,9 @@
              </a> [[!HTML5]]. The codecs parameter [[!RFC6381]], if necessary to
              specify exactly how the resource is encoded, SHOULD be added
              to <code>type</code>. If <var>type</var> can be expressed in several
-             equivalent ways (e.g. with or without quotes), the shortest form
-             and lowercase letters SHOULD be preferred.
+             equivalent ways (e.g. with or without quotes or whitespaces), the
+             one using the smallest amount of lowercase characters MUST be
```

Comment:
The "shortest form" bit was important. From the new wording "video/WebM" is better than "video/webm" because in fact it has fewer lowercase characters :)